test_data/yast/firstboot/yast2_firstboot.yaml

```diff
@@ -6,3 +6,6 @@ clients:
   - firstboot_user
   - firstboot_root
   - firstboot_finish
+settings:
```

Comment:
setting of which screen/client? we should nest them under a key reflecting that. But we can do this later (will be just a refactor) wdyt?

Reply:
This solution for comparing the language and keyboard settings, we can use it in all firstboot functions. For example, if in timezone, we push the current setting into %settings and we add what we expect in test_data-> settings, it will be also compared in function compare_settings.  This should also work with extra level of nesting but the only advantage is to be more comprehensible when someone is taking a look in test_data, which module  is setting each value, which is just a detail we don't really need. Since we have agreed to avoid extensive nesting when we can, I thought this way as more practical. If you think it's vital to separate the test_data, I can give it a try.

Reply:
we have nested structures everywhere, one level of nesting is ok to group each client IMO. But we can do it later when we have more fields, for now it is not needed, just to communicate to next developer (just reading the code) that to know which settings belong to each client would need some grouping.